test/e2e/framework/deployment.go

```diff
@@ -43,12 +43,28 @@ const HTTPBunService = "httpbun"
 // NipService name of external service using nip.io
 const NIPService = "external-nip"
 
+// HTTPBunImage is the default image that is used to deploy HTTPBun with the framwork
+const HTTPBunImage = "registry.k8s.io/ingress-nginx/e2e-test-httpbun:v20230505-v0.0.1"
+
+// EchoImage is the default image to be used by the echo service
+const EchoImage = "registry.k8s.io/ingress-nginx/e2e-test-echo@sha256:4938d1d91a2b7d19454460a8c1b010b89f6ff92d2987fd889ac3e8fc3b70d91a"
```

Comment:
Same as above. We can default this, but also get from env would be better

Reply:
I believe we will try remove echo soon so maybe not worth the effort here